mozfldp/simulation_util.py

```diff
@@ -23,23 +23,25 @@ def client_update(init_weights, epochs, batch_size, features, labels):
     batches_features = []
     batches_labels = []
 
+
     for i in range(0, len(features), batch_size):
         batches_features.append(features[i : i + batch_size])
         batches_labels.append(labels[i : i + batch_size])
 
     coef = list(init_weights[0])
     intercept = list(init_weights[1])
 
-    # set max_iter to 1 so that each .fit() call only does one training step
-    classifier = SGDClassifier(loss="hinge", penalty="l2", max_iter=1)
+    classifier = SGDClassifier(loss="log")
+    classifier.coef_ = np.array(coef)
+    classifier.intercept_ = np.array(intercept)
 
     for epoch in range(epochs):
         for i in range(len(batches_features)):
-            classifier.fit(
+            classifier.partial_fit(
```

Comment:
Just for my clarity - is 0-9 the complete list of all labels available in the MNIST dataset?